src/components/Dashboard/Form/Applicants/Applicants.test.js

```diff
@@ -20,22 +20,34 @@ describe("Applicants", () => {
   });
 
   it("renders without crashing", () => {
-    const div = document.createElement("div");
-    ReactDOM.render(
-      formWrapper(<Applicants reasonsList={[]} disabilityList={[]} />),
-      div
+    const div = create(
+      formWrapper(<Applicants reasonsList={[]} disabilityList={[]} />)
     );
-    ReactDOM.unmountComponentAtNode(div);
+    div.unmount();
   });
 
   it("contains select with first reason", () => {
-    const div = document.createElement("div");
-    ReactDOM.render(
-      formWrapper(<Applicants reasonsList={[]} disabilityList={[]} />),
-      div
+    const div = create(
+      formWrapper(<Applicants reasonsList={[]} disabilityList={[]} />)
     );
+    const applicants = div.root;
+    expect(
+      applicants.findByProps({ component: "select", name: "applicant.reason" })
+    ).toBeDefined();
+    div.unmount();
```

Comment:
You call this method every time after test and it is correct behavior but you make this every time manually. Maybe can we move `div` to root `describe` method and create a component ( in `beforeEach`) and unmount (in `afterEach`) for every test?